index.js

```diff
@@ -0,0 +1,31 @@
+const langSec = document.querySelector(".language-section");
```

Comment:
Abbreviation leads to confusion. It makes code less predictable and guessable.

`langSec` => `languageSelection`

When a variable contains an HTML Node, it is extremely common for developers to denote this with an `El` suffix, which is unfortunately an abbreviation, but it's one you need to learn. Stands for "element".

```suggestion
const languageSectionEl = document.querySelector(".language-section");
```